content/installation/netcore/install/System-requirements-netcore.md

```diff
@@ -0,0 +1,36 @@
+<!--
+title: "Contrast .NET Core Agent System Requirements"
+description: "Contrast .NET Core agent system requirements"
+tags: "installation agent .NET Core system requirements"
+-->
+
+## Before You Start 
+
+Before installing the .NET Core agent, confirm that you can meet the following requirements:
+
+- You have administrative access to a web server, and the server is supported by Contrast.
+- There is a deployed application to be analyzed, and the web application technology is supported by Contrast.
+- The web server has network connectivity with Contrast. 
+- The server meets the minimum requirements (stated below). 
+
+## Minimum Requirements
+
+* .NET Framework >= 4.7.0 installed
+* .NET Core >= 2.1 < 3.0 (installed system-wide)
```

Comment:
Is this supposed to mean "greater than or equal to"?

Reply:
Yep.